src/main/java/uk/gov/companieshouse/api/accounts/ResourceName.java

```diff
@@ -17,7 +17,8 @@
     FIXED_ASSETS_INVESTMENTS("fixed-assets-investments"),
     EMPLOYEES("employees"),
     CIC_REPORT("cic-report"),
-    CIC34_REPORT("cic34-report");
+    CIC34_REPORT("cic34-report"),
```

Comment:
same here - can we take out the `34` ?

Reply:
As above, this was pre-existing to Steve's changes and will be removed  shortly